votor/src/consensus_pool.rs

```diff
@@ -152,27 +149,18 @@ impl ConsensusPool {
         };
         let mut new_certificates_to_send = Vec::new();
         for cert_type in vote_to_certificate_ids(vote) {
-            // If the certificate is already complete, skip it
             if self.completed_certificates.contains_key(&cert_type) {
```

Comment:
unrelated to PR but just noticed: Does this imply we build at most 1 notar fallback cert (not 3)? If so, that would be a serious bug.